ctms/acoustic_service.py

```diff
@@ -236,11 +240,11 @@ def _newsletter_converter(self, acoustic_main_table, contact):
             if newsletter.name in AcousticResources.MAIN_TABLE_SUBSCR_FLAGS.keys():
                 newsletter_dict = newsletter.dict()
                 _today = datetime.date.today().isoformat()
-                newsletter_template["create_timestamp"] = self.dictionary_helper(
-                    newsletter_dict, "create_timestamp", _today
+                newsletter_template["create_timestamp"] = newsletter_dict.get(
```

Comment:
Thank you!

Reply:
I appreciated the call out, I was hitting my head; almost made a default_dict and ended up just recreating the extended `get` without realizing it. Using get with a default value...well I'm really happy that you corrected me so I can do this appropriately